### PR TITLE
Adds the pipelines metrics documentation

### PR DESCRIPTION
**Pull request by hrishin**
# Changes
Adds document about available metrics and its configuration options.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [X] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

Fixes https://github.com/tektoncd/pipeline/issues/1965

### CODE REVIEW

File: docs/metrics.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Pipeline Controller Metrics
+
+Following pipeline metrics exposed at `controller-service` on port `9090`
```
Comment (vdemeester) [inline docs/metrics.md]: The pipeline metrics are exposed at …

File: docs/metrics.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+Different kind of exporters as Prometheus, Google Stackdriver and so on could be configured using [observability configuration](../config/config-observability.yaml).
```
Comment (vdemeester) [inline docs/metrics.md]: There is different kind of exporters available : Prometheus, Google Stackdriver, etc. They can be configured using […].

File: docs/metrics.md (hunk 1 above)
Comment (hrishin) [reply docs/metrics.md]: :+1:

File: docs/metrics.md (hunk 2 above)
Comment (hrishin) [reply docs/metrics.md]: :+1:

**Comment by hrishin** [comment]
Thank you @vdemeester :) 

hi @afrittoli, could you please review it? :)

File: docs/metrics.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Pipeline Controller Metrics
+
+Following pipeline metrics are exposed at `controller-service` on port `9090`
+
+There are different kinds of exporters available: Prometheus, Google Stackdriver, etc. They can can be configured using [observability configuration](../config/config-observability.yaml). 
+
+| Metric name| Metric type | Labels/tags | Status |
+| ---------- | ----------- | ----------- | ----------- |
+| tekton_pipelinerun_duration_seconds_[bucket, sum, count] | Histogram | `pipeline`=&lt;pipeline_name&gt; <br> `pipelinerun`=&lt;pipelinerun_name&gt; <br> `status`=&lt;status&gt; <br> `namespace`=&lt;pipelinerun-namespace&gt; | beta |
```
Comment (afrittoli) [inline docs/metrics.md]: What does the status "beta" means in this context?

Comment (hrishin) [reply docs/metrics.md]: Well good point, I men't for more of experimental. Shall we call it out explicitly?

Thank you.

Comment (vdemeester) [reply docs/metrics.md]: @hrishin let's use `experimental` instead of `beta` :angel:

Comment (hrishin) [reply docs/metrics.md]: @vdemeester @afrittoli amended it :)

File: docs/metrics.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# Pipeline Controller Metrics
+
+Following pipeline metrics are exposed at `controller-service` on port `9090`
+
+There are different kinds of exporters available: Prometheus, Google Stackdriver, etc. They can can be configured using [observability configuration](../config/config-observability.yaml).
```
Comment (afrittoli) [inline docs/metrics.md]: sorry I didn't notice this earlier: "can can"

Comment (hrishin) [reply docs/metrics.md]: ah my bad rather, fixed :+1:

**Comment by hrishin** [comment]
@afrittoli @vdemeester addressed the review comments, thank you!

**Pull request merged**